### PR TITLE
Bundle tolerance

### PR DESCRIPTION
**Pull request by Rakunanthanks**
Added Scenario for Bundle Tolerance

### CODE REVIEW

**Review by vipul12agarwal — CHANGES_REQUESTED** [review]
In DefineBundleTolerence.java
1. Too many print statements are used
2. Page Object Model is not followed
3. Thread.sleep is used multiple times in one function, instead use explicit wait if required
4. For clearing and entering text, please use custom selenium command

File: src/test/resources/features/BundleTolerance.feature [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    And Checking the Bundle Tolerance Value
+    Then Check the Bundle Tolerance Value as per the Company Profile
+
+  @BundleTolerance
+  Scenario: Test Verifies the bundle tolerance by drawing Bundle,inspecting the bundle and changing the length of the bundle .
+    Given I'm on Arcadia test environment
+    And test data config loaded for test identifier test4
+    And Navigated to Test Project
+    And Bundles are drawn
+    And Inspecting Bundle
```
Comment (arpitbansalsre) [inline src/test/resources/features/BundleTolerance.feature:17]: These granular level of steps are not needed in feature file.

**Comment by arpitbansalsre** [comment]
Please rebase the code with main branch again without hampering the existing code.

File: src/test/java/arcadia/constants/EndPoint.java [hunk 2]
```diff
@@ -1,12 +1,20 @@
 package arcadia.constants;
 
 public enum EndPoint {
-    TRAINING("mercury1_21/"),
-    PROJECT("/index.lp?app=projects2&appoption=10&project=2&projName=projectName"),
+    TRAINING("mercury1_21_1/"),
+    PROJECT("/index.lp?app=projects2&appoption=10&project=2&projName=quickstart"),
```
Comment (arpitbansalsre) [inline src/test/java/arcadia/constants/EndPoint.java:5]: This change is not needed. The project name would be picked from environment variables.

**Pull request closed without merging**